tutorials/introductory/customizing.py

```diff
@@ -137,6 +137,12 @@ def plotting_function():
 #    >>> import matplotlib.pyplot as plt
 #    >>> plt.style.use('./images/presentation.mplstyle')
 #
+# You can include style sheets into standard importable Python packages (which
+# can be e.g. distributed on PyPI).  If your package is importable as
+# ``import my.package``, with a ``my/package/__init__.py`` module, and you add
+# a ``my/package/presentation.mplstyle`` style sheet, then it can be used as
+# ``plt.style.use("my.package.presentation")``.
```

Comment:
```suggestion
# ``import my.package``, with a ``mypackage/__init__.py`` module, and you add
# a ``mypackage/presentation.mplstyle`` style sheet, then it can be used as
# ``plt.style.use("mypackage.presentation")``.
```
As above. You may add an extra comment that nesting is supported.